tests/scripts/test_scripts.py

```diff
@@ -723,6 +723,29 @@ def test_train_rl_double_normalization(tmpdir: str, rng):
         )
 
 
+def test_train_rl_cnn_policy(tmpdir: str, rng):
+    venv = util.make_vec_env(
+        "AsteroidsNoFrameskip-v4",
+        n_envs=1,
+        parallel=False,
+        rng=rng,
+    )
+    print(venv)
+    net = reward_nets.CnnRewardNet(venv.observation_space, venv.action_space)
+    tmppath = os.path.join(tmpdir, "reward.pt")
+    th.save(net, tmppath)
+
+    log_dir_data = os.path.join(tmpdir, "train_rl")
+    with pytest.warns(RuntimeWarning):
```

Comment:
What warning are we expecting to see here? We don't have any warning specifically for this case do we?

(Separately I'm worried the other tests checking for `RuntimeWarning` are too vague, they might be passing because of other warnings that always get raised, we should probably add a match to check for the relevant message.)

Reply:
Yeah good point - removed checking for a warning here (doesn't really make sense). Should I make an issue for checking for the right messages on some of the other tests?

Reply:
Yes, please make an issue. If you could also make a PR to solve that issue that'd be great, but it's definitely out of scope for this PR.

Reply:
Looks like there's actually just one test that does this, made an issue at #614. Will put up a PR in a bit to address.